dev/io.openliberty.webcontainer.servlet.6.0.internal/src/io/openliberty/webcontainer60/osgi/request/IRequest60Impl.java

```diff
@@ -0,0 +1,41 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.webcontainer60.osgi.request;
+
+import com.ibm.websphere.ras.Tr;
+import com.ibm.websphere.ras.TraceComponent;
+import com.ibm.websphere.servlet40.IRequest40;
+import com.ibm.ws.webcontainer.osgi.osgi.WebContainerConstants;
+import com.ibm.ws.webcontainer40.osgi.request.IRequest40Impl;
+import com.ibm.wsspi.http.HttpInboundConnection;
+
+import io.openliberty.websphere.servlet60.IRequest60;
+
+
+/**
+ *
+ */
+public class IRequest60Impl extends IRequest40Impl implements IRequest60 {
+
```

Comment:
Is this class just crated in preparation of future dev work?

Reply:
It is needed to create request60 object  from the factory

https://github.com/OpenLiberty/open-liberty/pull/21525/files#diff-4335791aa89159df097f230ae82e239bd57323529f48b2bcd5bf7891c807f518